tests/run_tests.sh

```diff
@@ -25,6 +34,9 @@ for example in ${EXAMPLES}; do
         scp -o "StrictHostKeyChecking no" -P $SSHPORT "$BUILD_DIR/$example" root@127.0.0.1:/root
         exit_status=0
         RESULT={{$(ssh -o "StrictHostKeyChecking no" -p $SSHPORT -t root@127.0.0.1 "/root/$example 68")} && exit_status=1} || true
+        if [ "$example" = "read_only" ]; then
+            example="employee/""$example"
```

Comment:
Do you mean the first part ("employee/") or the second (surrounding $example)?

Reply:
I mean, why not just have `example=employee/$example` rather than all the `""`s?

Reply:
> I mean, why not just have `example=employee/$example` rather than all the `""`s?

I can fix that. It is a common practice to "quote everything" in POSIX scripting to avoid unexpected errors--especially for someone like me who forgets everything recently :/

Reply:
I've never seen *that* level of quoting before :p

Reply:
Fixed in d133748.